physcraper/__init__.py

```diff
@@ -3597,6 +3387,44 @@ def replace_new_seq(self):
 
 
 ####################
+#Funcs below here should be moved to ncbi_helpers.py
```

Comment:
Currently, the ncbi_helper class is a class that stores the ncbi taxonomy database files to be accessed by functions which are using a local blast database. The class is not present for the web version, as those files are not present there.

Reply:
Oh ya, my comment was unclear. I thought they should probably get pulled out to a different file, of naming helpers, but not within that class. But you're right that putting them in that file would probably  confuse things.